make.bat

```diff
@@ -9,14 +9,15 @@ rem nmake
 rem meson configure --buildtype=release ..\src
 rem CD ..\release
 rem ninja -v -j4
-CD src && meson ..\build
+CD src && meson --buildtype=release ..\build
 CD ..\build && ninja -j4
 REM IF !ERRORLEVEL! NEQ 0 EXIT /B 1
 CD ..
 
 ECHO Deploying iaito
 MKDIR iaito
 COPY build\iaito.exe iaito\iaito.exe
+RMDIR /S /Q build

```

Comment:
uhm i dont think we want to remove this directory, as make.bat is not only used by the CI, and local builds use to require build+src modification + build, and removing this directory will force a full recompilation every time

Reply:
Didn't think about that, but leaving that directory causes errors in CI e.g., in [w64-meson's third upload artifact](https://github.com/FetheredSerpent/iaito/actions/runs/6138807580/job/16656043288), which I was unable to understand. Simply deleting the 'build' directory, fixes it 😅 but leads to full recompilation. Don't know what to do to fix it.

Also, currently there are several build directories, one named ['build_%PLATFORM%'](https://github.com/radareorg/iaito/blob/77be9b82c1eb0bdd4f196fa9ff3ea597402a78cb/configure.bat#L24) for qmake, ['build'](https://github.com/radareorg/iaito/blob/77be9b82c1eb0bdd4f196fa9ff3ea597402a78cb/.github/workflows/ci.yml#L219C9-L219C9) which the CI is uploading as artifact and ['release'](https://github.com/radareorg/iaito/blob/77be9b82c1eb0bdd4f196fa9ff3ea597402a78cb/make.bat#L8) which is used by meson. I have renamed all to 'build_%PLATFORM%' by constantly deleting it after every step (which is very bad as stated above). So, which directories should I be keeping? I couldn't find any reference to 'build' (for Windows) except the CI, therefore maybe we can delete it and keep the other two, or is the 'release' used by meson a typo? Should I leave the directories as they are?

Reply:
The release is all whats published so i consider it a destdir or pubdir. About the builddir i think there should be only one like it happens on meson/make on unix. Even if qmake is later on doing its thing i think the folder can be the same.

but also if we want to remove a builddir this should happen when running "make.bat clean". So this can be added as an extra step in the ci if needed. So the local dev workflow doesnt gets altered by this

how does it sound?

Reply:
Sounds good, I'll try to unify the build directories then

Reply:
Btw in ci.yml, is [env.PACKAGE_NAME](https://github.com/radareorg/iaito/blob/77be9b82c1eb0bdd4f196fa9ff3ea597402a78cb/.github/workflows/ci.yml#L218) predefined or accidently undeclared?